api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -671,6 +671,36 @@ private boolean userNeverLoggedIn(com.google.api.services.admin.directory.model.
   @Override
   @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
   public ResponseEntity<EmptyResponse> bypassAccessRequirement(Long userId, String moduleName, AccessBypassRequest request) {
+    updateBypass(userId, moduleName, request);
+    return ResponseEntity.ok(new EmptyResponse());
+  }
+
+  @Override
+  public ResponseEntity<EmptyResponse> unsafeSelfBypassAccessRequirement(String moduleName, AccessBypassRequest request) {
+    if (!workbenchConfigProvider.get().access.unsafeAllowSelfBypass) {
+      throw new ForbiddenException("Not allowed to do self bypass.");
+    }
+    long userId = userProvider.get().getUserId();
+    updateBypass(userId, moduleName, request);
```

Comment:
Indirectly related: I just noticed there's a module called "idVerification" defined in `updateBypass`. Is this old code - or is this speculative for future IDV? Either way I think we should probably not be allowing bypass of it today.

Reply:
I can remove the bypass on it.